### PR TITLE
Refactor how commands are run on the client

### PR DESCRIPTION
**Pull request by brandonsturgeon**
It feels dangerous and a little irresponsible to be telling the client what commands to run each time.

We can just tell them what commands to run when it's time for a cleanup.

### CODE REVIEW

File: lua/autorun/client/cl_autoclean.lua [hunk 1]
```diff
@@ -8,11 +8,8 @@ local function notifyPlayer( notification )
     print( message )
 end
 
-net.Receive( "CFC_AutoClean_RunCommand", function() 
-    local command = net.ReadString()
-
-    if isAllowedCommand[command] then
-        notifyPlayer( "The server is running '" .. command .. "' on your client.." )
+net.Receive( "CFC_RunAutoClean", function() 
+    for command, _ in pairs(cleanupCommands) do
```
Comment (PLally) [inline lua/autorun/client/cl_autoclean.lua]: would an array of commands be better now because the true value doesn't change anything?

Comment (brandonsturgeon) [reply lua/autorun/client/cl_autoclean.lua]: what does the loop look like for running through an array again?

Comment (PLally) [reply lua/autorun/client/cl_autoclean.lua]:
```lua
for _, k in pairs(r) do

end
```

Comment (PLally) [reply lua/autorun/client/cl_autoclean.lua]: or
```lua
for i=1, #r do
    local k = r[i]
end
```

**Comment by brandonsturgeon** [comment]
@PLally ping

File: lua/autorun/server/sv_autoclean.lua [hunk 2]
```diff
@@ -20,11 +15,8 @@ local function notifyPlayers( notification )
 end
 
 local function runCleanupCommandsOnPlayers()
-    for command, _ in pairs( clientCleanupCommands ) do
-        net.Start( "CFC_AutoClean_RunCommand" )
-        net.WriteString( command )
-        net.Broadcast()
-    end
+    net.Start( "CFC_RunAutoClean" )
```
Comment (PLally) [inline lua/autorun/server/sv_autoclean.lua]: you need to update  `util.AddNetworkString( "CFC_AutoClean_RunCommand" )
other than that it looks good 👍

**Comment by brandonsturgeon** [comment]
retest this please

**Comment by brandonsturgeon** [comment]
retest this please

**Comment by brandonsturgeon** [comment]
Reopening as a test

**Pull request closed without merging**